qcfractal/interface/client.py

```diff
@@ -82,6 +83,27 @@ def __init__(self,
 
         self.server_name = self.server_info["name"]
 
+        def version_list(version):
```

Comment:
Could we move this above the class with a `_version_list` name? We might want to move this to a util eventually.